3493/0/about_classes.rb

```diff
@@ -0,0 +1,189 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+#:reek:FeatureEnvy:reek:TooManyMethods:reek:TooManyStatements:reek:InstanceVariableAssumption:
+class AboutClasses < Neo::Koan
+  class Dog
+  end
+
+  def test_instances_of_classes_can_be_created_with_new
+    fido = Dog.new
+    assert_equal Dog, fido.class
+  end
+
+  # ------------------------------------------------------------------
+  # rubocop: disable Naming/AccessorMethodName,
+  # rubocop: disable Style/EvalWithLocation, Lint/AmbiguousBlockAssociation
+  class SecondDog
```

Comment:
**FYI**
Bad naming. In real project you've get many files like this:
```
dog.rb
second_dog.rb
six_dog.rb
similar_cat.rb
```

The best way to get files from folder will be:
```
dog.rb
dog_second.rb
dog_six.rb
cat_similar.rb
```
So you will have `dog` like namespace